grpc-api/src/main/java/io/deephaven/grpc_api/session/ExportTicketResolver.java

```diff
@@ -25,27 +25,29 @@ public ExportTicketResolver() {
     }
 
     @Override
-    public String getLogNameFor(ByteBuffer ticket) {
-        return ExportTicketHelper.toReadableString(ticket);
+    public String getLogNameFor(final ByteBuffer ticket, final String logId) {
+        return ExportTicketHelper.toReadableString(ticket, logId);
     }
 
     @Override
-    public SessionState.ExportObject<Flight.FlightInfo> flightInfoFor(@Nullable final SessionState session,
-            final Flight.FlightDescriptor descriptor) {
+    public SessionState.ExportObject<Flight.FlightInfo> flightInfoFor(
+            @Nullable final SessionState session, final Flight.FlightDescriptor descriptor, final String logId) {
         if (session == null) {
-            throw GrpcUtil.statusRuntimeException(Code.UNAUTHENTICATED, "No session to search");
+            throw GrpcUtil.statusRuntimeException(Code.UNAUTHENTICATED,
+                    "Could not resolve '" + logId + "': no exports can exist without a session to search");
```

Comment:
'search' seems like a weird way to describe this operation, why not just "no exports can exist without a session" or "without a valid session", "active session" etc?